src/components/elements/table/GenericTable.tsx

```diff
@@ -247,7 +243,12 @@ const GenericTable = <T extends { id: string }>({
                 width: def.width,
               }}
             >
-              <strong>{def.header}</strong>
+              {def.header ? (
```

Comment:
Continuing the thread from [here](https://github.com/greenriver/hmis-frontend/pull/989#discussion_r1900164832):

> {def.key || def.someOtherFieldThatsRequiredIfHeaderIsMissing}

If you agree, I was thinking that based on the comment [here](https://github.com/greenriver/hmis-frontend/pull/998/files#diff-e192fff2380d8eba4e69c51f1fcb73a656815e79a902758b40ce626c9c304da5L12),
> unique key for element. if not provided, header is used

We probably don't really need to add an extra `someOtherFieldThatsRequiredIfHeaderIsMissing`, because `key` should serve exactly that purpose.

Reply:
Nice -- sounds good, I see that with the union types you achieved that required-ness of `header` or `key`. Looks good!